3rd pull, Microverse Code review pull

AmineHLub opened the pull request:
- Add background urls in css

- Fix navbar margin

- Combined Social section as a DIV inside the first mainpage section (html)

AmineHLub committed:
Add background urls in css

Fix navbar margin

Combined Social section as a DIV inside the first mainpage section (html)

addod19 commented on index.html:
```diff
@@ -20,13 +20,13 @@
-        <a class="mediumlogo" href="#"><img src="./src/medium.png" alt="medium"></a>
-        <a href="#"><img src="./src/github.png" alt="github"></a>
-        <a href="#"><img src="./src/angellist.png" alt="angellist"></a>
+        <div id="socials">
```
- [ ] It would be better if we change `<div id="socials">` to `<ul id="socials">` to improve HTML semantics

addod19 commented on index.html:
```diff
@@ -20,13 +20,13 @@
-        <a href="#"><img src="./src/angellist.png" alt="angellist"></a>
+        <div id="socials">
+        <a class="logos" href="#"><img src="./src/twitter.png" alt="twitter"></a>
+        <a class="logos" href="#"><img src="./src/linkedin.png" alt="linkedin"></a>
+        <a class="logos mediumlogo" href="#"><img src="./src/medium.png" alt="medium"></a>
+        <a class="logos" href="#"><img src="./src/github.png" alt="github"></a>
+        <a class="logos" href="#"><img src="./src/angellist.png" alt="angellist"></a>
```
- [ ] It would be better if we change `<a class="logos">` to `<li class="logos">` to improve HTML semantics

AmineHLub replied:
but the li does not have href attribute, so I will insert the hyperlink a inside the li

addod19 commented on style.css:
```diff
@@ -18,9 +18,6 @@ header {
   justify-content: space-between;
   align-items: center;
   text-align: center;
```
- [ ] It would be better if we implement the CSS DRY principle, there are a few repeatings rules that need to be cleaned. For example;
```
.nav-container, .mainpage {
  display: flex;
  align-items: center;
  text-align: center;
}
```

addod19 commented on index.html:
```diff
@@ -20,13 +20,13 @@
+        <a class="logos" href="#"><img src="./src/angellist.png" alt="angellist"></a>
+        </div>
     </section>
     <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
```
### Optional
- It would be better not to have a lot of `<br>` tags, it is better to create an empty div and give the spacing you desire

AmineHLub replied:
This space is only temporary to check the sticky navbar however I'll remove them right away or change them to an empty div. Thanks!

addod19 commented on style.css:
```diff
@@ -72,11 +78,15 @@ h1 {
+  margin-top: 0.8vh;
+}
+
+.logos {
```
- [x] It would be better if we use the recommended color for our social media logos, here is the recommended color `#505F79`

AmineHLub replied:
That's an actual wrong css mistake by me there is as a matter of fact no need for a color at all i was checking if could make the background was not transparent but then i realised it was just that items were not behind a box but instead not in the same position. If I apply any color the "logos" box will not match the given template. So sorry for this

addod19 reviewed (changes requested):
### Hi @AmineHLub , 

Good job so far! Git flow followed :muscle: PR and README very professional :fireworks: 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

__Happy coding__ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

AmineHLub committed:
Replace div to ul

Insert anchor hyperlink a inside li

Replace br with temporary spacer div

Implement CSS DRY

Remove unecessary css background from logos class

AbdumurodovaZulfizar commented on style.css:
```diff
@@ -1,31 +1,39 @@
-  background-color: #e5e5e5;
-}
-
 header {
+  background-color: azure;
```
- [ ] **REQUIRED:** Please follow the design given on **Figma**, it is actually white, and for your page the different color making the UI/UX a bit ugly.
![image](https://user-images.githubusercontent.com/77872355/129299077-3332edd3-cb3c-4dc8-9220-c1fc94850890.png)

AmineHLub replied:
I did it like that so that I have a sticky navbar on top, but I removed it now

AbdumurodovaZulfizar replied:
But as you can see the colour is not covering full-width, and it is better to follow the design because UI/UX specialists created these templates according to modern software demand.

AbdumurodovaZulfizar commented on index.html:
```diff
@@ -20,15 +20,15 @@
         <h1 class="name">Hey there. I’m<br>Mariane</h1>
```
- [ ] **REQUIRED:** There is a bit much more space between header _(I mean toolbar)_ and section, kindly add space by adding margin or padding to `<h1>` element.

- [ ] **OPTIONAL:** Please make the hamburger button a bit smaller, try to make the same size the portfolio you have chose.

AbdumurodovaZulfizar commented on style.css:
```diff
@@ -34,19 +42,23 @@ header {
 }
 
 .mainpage {
-  display: flex;
   flex-flow: column wrap;
-  align-items: center;
-  text-align: center;
+  background-image: url(./bg/9.png), url(/bg/3.png), url(bg/1.png), url(bg/8.png), url(bg/4.png), url(/bg/7.png);
+  background-repeat: no-repeat;
```
- [ ] **REQUIRED:** Your background image is not covering the full width, there is little space on both right and left side, please kindly set the width 100wh.

AbdumurodovaZulfizar commented on index.html:
```diff
@@ -20,15 +20,15 @@
         <h1 class="name">Hey there. I’m<br>Mariane</h1>
         <h1 class="detail">I’m a software developer</h1>
         <p class="pOne">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```
- [x] **OPTIONAL:** The paragraph should have the same padding on left and right side with the logo and hamburger button, please fix it.

AbdumurodovaZulfizar commented on index.html:
```diff
@@ -20,15 +20,15 @@
-    <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
+    <div class="temporary-spacer-div"></div>
 
 
 </body>
```
- [ ] **REQUIRED:** The given live demo link on the README.md file is leading to another branch's changes, I offer you to check the GitHub pages.

- [ ] **REQUIRED:** Kindly remove the unused parts from README.md like installation and running tests( It is not a test to check your project, actually you will do testing on other languages), and it would be good if you remove `Internet connection` from prerequisites, instead of it add code editor or live server.

AbdumurodovaZulfizar reviewed (changes requested):
Hello @AmineHLub, I am @AbdumurodovaZulfizar, your reviewer for this project :wave:

Congratulations on doing an awesome project and thanks for implementing the required changes by the previous reviewer💥
**TO HIGHLIGHT:**

- You have set up linters correctly and there are no errors
- You have written descriptive comments with the correct tense
- You have used `flex` to align elements

But, to master your work, there is some work to do, I offer you to try to make the design the same and pay attention to the live demo link.

### **CHANGES REQUIRED ♻️ 🛑**


**Look through the comments below 👇 for required changes**


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
<hr>

Happy Coding 🔥🔥🔥
The review was done by @AbdumurodovaZulfizar
If there is anything that is not 💯% clear don't hesitate to hit me on slack @Zulfizar Abdumurodova or leave messages on PR.

<hr>

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AmineHLub committed:
Remove sticky nav

Replace body background to the one of figma regardless of look

fix h1 spacing with navbar

Make hamburger menu smaller

fix background(make it inside the body and not the element)

Fix navbar paddings to match with the rest of the page

fix readme for unecessary given data

Change live link from master to current branch

AmineHLub committed:
Fix padding between social media logos

mke2111 reviewed (approved):
### Hi @AmineHLub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
